mirrord/cli/src/operator/status.rs

```diff
@@ -0,0 +1,306 @@
+            if copy_targets.is_empty() {
+                println!("No active copy targets.");
+            } else {
+                println!("Active Copy Targets:");
+                let mut copy_targets_table = Table::new();
+
+                copy_targets_table.add_row(row![
+                    "Original Target",
+                    "Namespace",
+                    "Copy Pod Name",
+                    "Scale Down?"
+                ]);
+
+                for (pod_name, copy_target_resource) in copy_targets {
+                    copy_targets_table.add_row(row![
+                        copy_target_resource.spec.target.to_string(),
+                        copy_target_resource
+                            .metadata
+                            .namespace
+                            .as_deref()
+                            .unwrap_or_default(),
+                        pod_name,
+                        if copy_target_resource.spec.scale_down {
+                            "*"
+                        } else {
+                            ""
+                        },
+                    ]);
+                }
+
+                copy_targets_table.printstd();
+            }
+            println!();
+        }
+
+        if let Some(statistics) = status.statistics.as_ref() {
+            println!("Operator Daily Users: {}", statistics.dau);
+            println!("Operator Monthly Users: {}", statistics.mau);
+        }
+
+        let mut sessions = Table::new();
+
+        sessions.add_row(row![
+            "Session ID",
+            "Target",
+            "Namespace",
+            "User",
+            "Ports",
+            "Session Duration"
+        ]);
+
+        let mut sqs_rows = None;
+        for session in &status.sessions {
+            let locked_ports = session
+                .locked_ports
+                .as_deref()
+                .map(|ports| {
+                    ports
+                        .iter()
+                        .map(|(port, type_, filter)| {
+                            format!(
+                                "Port: {port}, Type: {type_}{}",
+                                filter
+                                    .as_ref()
+                                    .map(|f| format!(", Filter: {}", f))
+                                    .unwrap_or_default()
+                            )
+                        })
+                        .collect::<Vec<_>>()
+                        .join("\n")
+                })
+                .unwrap_or_default();
+
+            sessions.add_row(row![
+                session.id.as_deref().unwrap_or_default(),
+                &session.target,
+                session.namespace.as_deref().unwrap_or("N/A"),
+                &session.user,
+                locked_ports,
+                humantime::format_duration(Duration::from_secs(session.duration_secs)),
+            ]);
+
+            sqs_rows = session.sqs.as_ref().and_then(|sqs| {
```

Comment:
this overrides `sqs_rows` with every session, no? So when you add the rows later you're only adding the rows of the last session.

Reply:
Yeah, it should've been `extend`. Changed.